pgModel/models.go

```diff
@@ -0,0 +1,64 @@
+package pgModel
+
+type JwtToken struct {
+    Token string `json:"token"`
+}
+
+type Exception struct {
+    Message string `json:"message"`
+}
+
+type User struct {
+	Id     int64 `sql:",pk,unique"`
+	Username   string
+	Email string `sql:",unique,notnull"`
+	Password string
+	PhoneNumber string
+	Messages []*Message
+	Connections []*Connection
+	Comments []*Comment
+	Collections []*Collection
+}
+
+type Connection struct {
+	Id int64
+	FirstUserId int64
+	SecondUserId int64
+}
+
+type Message struct {
+	Id int64
+	UserId int64
```

Comment:
@mikey2020 can we call this SenderId instead?
So as to match with the `ReceiverId` naming

Reply:
ok